packages/web/src/devtools.rs

```diff
@@ -237,3 +242,26 @@ pub(crate) fn invalidate_browser_asset_cache() {
         }
     }
 }
+
+/// Initialize required devtools for dioxus-playground.
+///
+/// This listens for window message events from other Windows (such as window.top when this is running in an iframe).
+#[cfg(feature = "devtools-playground")]
+fn playground(tx: UnboundedSender<HotReloadMsg>) {
+    let window = web_sys::window().unwrap();
+    window.set_onmessage(Some(
```

Comment:
we should also call the original onmessage if there was one

Reply:
Do you mean to check if there is already a `onmessage` callback registered and to call it from inside our callback?